datalad/plugin/dlplugin_add_readme.py

```diff
@@ -0,0 +1,79 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#PLUGINSYNOPSIS: add a README file to a dataset
+"""
+
+"""
+
+__docformat__ = 'restructuredtext'
+
+
+# PLUGIN API
+def datalad_plugin(dataset, filename='README.rst', existing='skip'):
```

Comment:
I wondered if we really need two levels of different "identification prefixes" -- "dlplugin_" for the filename and "datalad_" for the function?  imho should be enough to have just "plugin" here